catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.kt

```diff
@@ -0,0 +1,767 @@
+        motionBrickList.add(TurnRightSpeedBrick(BrickValues.PHYSIC_TURN_DEGREES))
+        motionBrickList.add(SetGravityBrick(BrickValues.PHYSIC_GRAVITY))
+        motionBrickList.add(SetMassBrick(BrickValues.PHYSIC_MASS))
+        motionBrickList.add(SetBounceBrick(BrickValues.PHYSIC_BOUNCE_FACTOR * BrickValues.PHYSIC_MULTIPLIER))
+        motionBrickList.add(SetFrictionBrick(BrickValues.PHYSIC_FRICTION * BrickValues.PHYSIC_MULTIPLIER))
+        if (SettingsFragment.isPhiroSharedPreferenceEnabled(context)) {
+            motionBrickList.add(PhiroMotorMoveForwardBrick(PhiroMotorMoveForwardBrick.Motor.MOTOR_LEFT, BrickValues.PHIRO_SPEED))
+            motionBrickList.add(PhiroMotorMoveBackwardBrick(PhiroMotorMoveBackwardBrick.Motor.MOTOR_LEFT, BrickValues.PHIRO_SPEED))
+            motionBrickList.add(PhiroMotorStopBrick(PhiroMotorStopBrick.Motor.MOTOR_BOTH))
+        }
+        return motionBrickList
+    }
+
+    protected open fun setupSoundCategoryList(context: Context): List<Brick> {
+        val soundBrickList: MutableList<Brick> = ArrayList()
+        soundBrickList.add(PlaySoundBrick())
+        soundBrickList.add(PlaySoundAndWaitBrick())
+        soundBrickList.add(StopSoundBrick())
+        soundBrickList.add(StopAllSoundsBrick())
+        soundBrickList.add(SetVolumeToBrick(BrickValues.SET_VOLUME_TO))
+        soundBrickList.add(ChangeVolumeByNBrick(Formula(BrickValues.CHANGE_VOLUME_BY)))
+        if (SettingsFragment.isAISpeechSynthetizationSharedPreferenceEnabled(context)) {
+            soundBrickList.add(SpeakBrick(context.getString(R.string.brick_speak_default_value)))
+            soundBrickList.add(SpeakAndWaitBrick(context.getString(R.string.brick_speak_default_value)))
+        }
+        if (SettingsFragment.isPhiroSharedPreferenceEnabled(context)) {
+            soundBrickList.add(PhiroPlayToneBrick(PhiroPlayToneBrick.Tone.DO, BrickValues.PHIRO_DURATION))
+        }
+        if (SettingsFragment.isAISpeechRecognitionSharedPreferenceEnabled(context)) {
+            soundBrickList.add(AskSpeechBrick(context.getString(R.string.brick_ask_speech_default_question)))
+            soundBrickList.add(StartListeningBrick())
+            soundBrickList.add(SetListeningLanguageBrick())
+        }
+        soundBrickList.add(SetInstrumentBrick())
+        soundBrickList.add(PlayNoteForBeatsBrick(BrickValues.DEFAULT_NOTE, BrickValues.PAUSED_BEATS_INT))
+        soundBrickList.add(PlayDrumForBeatsBrick(BrickValues.PAUSED_BEATS_INT))
+        soundBrickList.add(SetTempoBrick(BrickValues.DEFAULT_TEMPO))
+        soundBrickList.add(ChangeTempoByNBrick(BrickValues.CHANGE_TEMPO))
+        soundBrickList.add(PauseForBeatsBrick(BrickValues.PAUSED_BEATS_FLOAT))
+        return soundBrickList
+    }
+
+    protected open fun setupLooksCategoryList(context: Context, isBackgroundSprite: Boolean): List<Brick> {
+        val looksBrickList: MutableList<Brick> = ArrayList()
```

Comment:
can be simplified as 
```suggestion
        val looksBrickList = mutableListOf<Brick>()
```